### ISSUE TITLE
.ply import dialog: button to add all properties in PLY file as new scalar fields

### ISSUE THREAD

**Issue by eudoxos** [CloudCompare/CloudCompare#782]
## Current Behavior
When opening .ply file with a number of properties per point, each property has to be assigned to a new scalar field. This is a lot of manual work.

It would be nice to have one button to assign all well-known peroperties to their respective fields (like x,y,z, nx,ny,nz, ... -- this is done already) and create new scalar fields for all remaining properties.

## Steps to Reproduce (for bugs)

Open any .ply file with a number of per-point properties defined. It takes a while to assign all of them to scalar field for each property, -- (a) click "Add scalar field" (b) find the property in the new scalar drop-down. 

## Your Environment
* CC Version: 2.10alpha (edge snap channel)
* OS & Version: Ubuntu 18.04 LTS 
* Graphics card: AMD FirePro V7900

**Comment by softyoda** [comment]
I agree, i don't know how to not spend 15min each ply i have to open 
![image](https://github.com/user-attachments/assets/1c6c1e98-63fc-4cc0-bc18-692f59d39965)

**Comment by dgirardeau** [comment]
Ok, I gave this a try: see commit ef83f48a14b2b33e70efc7156df18b7601d85fec

You now have an 'Add all' button next to the 'Add' one.

You can test it in the latest 2.14.alpha version (on Windows): https://www.cloudcompare.org/release/CloudCompare_v2.14.alpha_setup_x64.exe
